docs/leverage.md

```diff
@@ -1,21 +1,81 @@
 # Leverage
+*You can't run 2 bots on the same account with leverage,*
+
+!!! Warning: Trading with leverage(`trading_mode="margin"` or `trading_mode="futures"`) is very risky. 
```

Comment:
What's the actual way I should give a big warning? This is failing a test. I pulled the !!! from somewhere else in the freqtrade docs

Reply:
The syntax for the  "info" / warning / ...  boxes can be found [here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) - we also link to it from the [developer docs](https://www.freqtrade.io/en/stable/developer/#documentation)

the syntax is basicaly `!!! Warning ["Optional header"]` (no colon!) - followed by the content, indented by 4 spaces.

Reply:
is `"Optional header"` literally in quotations?

Reply:
in quotes - yes.
We're using this in "some" places - searching for `!!! Note ` (note the space at the end) should yield some examples. Looking at the rendered documentation then, you can see the outcome (or use `mkdocs serve` locally)